src/app/modules/configuration/configuration.module.ts

```diff
@@ -1,11 +1,10 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { MatDialogModule, MatRadioModule, MatButtonModule, MatCardModule } from '@angular/material';
-import { SharedModule } from 'src/app/shared/shared.module';
-import { CoreModule } from 'src/app/core';
+import { SharedModule ,DynamicFormComponent } from 'shikshalokam';
+import { CoreModule } from 'shikshalokam';
```

Comment:
Its already included in app module. do we need to import it in other modules also?